test/test_environment_variables.rb

```diff
@@ -9,11 +9,11 @@
 class TestEnvironmentVariables
   module Common
     TEST_PLATFORM = ENV["TEST_PLATFORM"] || "x64-mingw-ucrt"
-    DOCKERHUB_USER = ENV['DOCKERHUB_USER'] || "larskanis"
+    DOCKER_REGISTRY = ENV['DOCKERHUB_USER'] || ENV['DOCKER_REGISTRY'] || "ghcr.io/rake-compiler"
```

Comment:
I don't think we need to preserve `DOCKERHUB_USER`. So far, it was only used within the project, not by the gem.